src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/activities/ActivitiesMigrationService.kt

```diff
@@ -50,15 +50,19 @@ class ActivitiesMigrationService(
     migrationFilter: ActivitiesMigrationFilter,
     pageSize: Long,
     pageNumber: Long,
-  ): PageImpl<FindActiveActivityIdsResponse> = nomisApiService.getActivityIds(
-    prisonId = migrationFilter.prisonId,
-    courseActivityId = migrationFilter.courseActivityId,
-    pageNumber = pageNumber,
-    pageSize = pageSize,
-  )
+  ): PageImpl<ActivitiesMigrationRequest> {
+    val start = migrationFilter.activityStartDate ?: LocalDate.now().plusDays(1)
```

Comment:
Default to starting migrated activities tomorrow. This matches current behaviour allowing us to go live before the UI changes are implemented.